flocx_market/db/sqlalchemy/api.py

```diff
@@ -73,6 +80,7 @@ def offer_get_all_by_status(status, context):
 def offer_create(values, context):
 
     values['marketplace_offer_id'] = uuidutils.generate_uuid()
+    values['project_id'] = context.project_id
```

Comment:
It turns out we don't want this; a marketplace offer is created by the provider, and we just want the provider to pass in the appropriate project_id from its own offer.

Reply:
But we should keep the context right? Just also take in the extra parameter?

Reply:
And when you say the provider should pass it in, are you saying the provider should pass it into the rest api?

Reply:
So I can just remove this line and the fix it good?

Reply:
But this line does not need to be removed from the bids as the bids are made directly with the marketplace, as opposed to going through a provider. Right?

Reply:
Yep, just remove this line and you're good. And you're exactly correct about the bids!